backend/etcd/etcd.go

```diff
@@ -112,6 +112,11 @@ type Etcd struct {
 	loopbackURL string
 }
 
+// BackendID returns the ID of the etcd cluster member
+func (e *Etcd) BackendID() (result string) {
+	return fmt.Sprintf("%s", e.etcd.Server.ID())
```

Comment:
should use String() instead of fmt.Sprintf

Reply:
Oh snap, golangci